clients/admin-ui/src/features/common/DaysLeftTag.tsx

```diff
@@ -1,13 +1,22 @@
 import { Tag } from "@fidesui/react";
 import React from "react";
 
+import { PrivacyRequestStatus } from "~/types/api/models/PrivacyRequestStatus";
```

Comment:
I used the autogenerated types version of this type because it was made it easier to do the comparison check. The original type was done like this 

```typescript
export type PrivacyRequestStatus =
  | "approved"
  | "complete"
  | "denied"
  | "error"
  | "in_processing"
  | "paused"
  | "canceled"
  | "pending"
  | "identity_unverified"
  | "requires_input";
```

This method doesn't allow you do to `PrivacyRequestStatus.COMPLETE` style comparisons. In the future we may want to convert everything to use the api version.

Reply:
ooooh yes we should definitely convert everything to use the API version, thanks for fixing this one!